lib/runtime/src/storage/key_value_store.rs

```diff
@@ -221,10 +239,10 @@ impl KeyValueStoreManager {
         cancel_token: CancellationToken,
     ) -> (
         tokio::task::JoinHandle<Result<(), StoreError>>,
-        tokio::sync::mpsc::UnboundedReceiver<WatchEvent>,
+        tokio::sync::mpsc::Receiver<WatchEvent>,
```

Comment:
Wondering if you have seen any issue with the `UnboundedReceiver`, which motivated the switch to `Receiver`?